src/game/AI/ScriptDevAI/scripts/eastern_kingdoms/naxxramas/boss_anubrekhan.cpp

```diff
@@ -65,6 +65,12 @@ struct boss_anubrekhanAI : public ScriptedAI
         m_instance = (instance_naxxramas*)creature->GetInstanceData();
         m_introDialogue.InitializeDialogueHelper(m_instance);
         m_hasDoneIntro = false;
+        m_creature->GetCombatManager().SetLeashingCheck([&](Unit*, float /*x*/, float /*y*/, float)
+        {
+            float x, y, z;
+            m_creature->GetRespawnCoord(x, y, z);
```

Comment:
There is a DB field called "leash" that does exactly this instead of needing code. The hook is to be used in non-circular leash cases. Also you are missing the /*z*/ there :P

Reply:
I'll update the scripts accordingly. Thank you! :)

Reply:
@killerwife the bosses all have leash values of 0 in the database, so that requires a DB update. Where should I put the update SQL file? In the tbc-db repository under Updates, or somewhere in the mangos-tbc repo?

Reply:
tbc-db is the place. classic-db will pull it then from there.